contracts/src/main/kotlin/com/r3/corda/lib/reissuance/contracts/ReissuanceLockContract.kt

```diff
@@ -247,14 +246,14 @@ class ReissuanceLockContract<T>: Contract where T: ContractState {
         val availableComponentNonces: Map<Int, List<SecureHash>> by lazy {
             wireTransaction.componentGroups.map { Pair(it.groupIndex, it.components.mapIndexed {
                 internalIndex, internalIt ->
-                componentHash(internalIt, wireTransaction.privacySalt, it.groupIndex, internalIndex) })
+                digestService.componentHash(internalIt, wireTransaction.privacySalt, it.groupIndex, internalIndex) })
```

Comment:
Use DigestService.default
Save having to create a new object.

Reply:
Done.